library/swift

```diff
@@ -6,55 +6,109 @@ Maintainers: Ted Kremenek <kremenek@apple.com> (@tkremenek),
+
+Tags: 5.10.0-centos7-slim, 5.10-centos7-slim, centos7-slim
+GitCommit: ea035798755cce4ec41e0c6dbdd320904cef0421
+Directory: 5.10/centos/7/slim
+
+Tags: 5.10.0-rhel-ubi9, 5.10-rhel-ubi9, rhel-ubi9
+Architectures: amd64, arm64v8
+GitCommit: ea035798755cce4ec41e0c6dbdd320904cef0421
+Directory: 5.10/rhel-ubi/9
+
+Tags: 5.10.0-rhel-ubi9-slim, 5.10-rhel-ubi9-slim, rhel-ubi9-slim
+Architectures: amd64, arm64v8
+GitCommit: ea035798755cce4ec41e0c6dbdd320904cef0421
+Directory: 5.10/rhel-ubi/9/slim
+
+Tags: 5.10.0-windowsservercore-ltsc2022, 5.10-windowsservercore-ltsc2022, windowsservercore-ltsc2022
+Architectures: windows-amd64
+GitCommit: ea035798755cce4ec41e0c6dbdd320904cef0421
+Directory: 5.10/windows/LTSC2022
+Constraints: windowsservercore-ltsc2022
+
+Tags: 5.9.2, 5.9, 5.9.2-jammy, 5.9-jammy
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/ubuntu/22.04
 
-Tags: 5.9.2-slim, 5.9-slim, 5.9.2-jammy-slim, 5.9-jammy-slim, jammy-slim, slim
+Tags: 5.9.2-slim, 5.9-slim, 5.9.2-jammy-slim, 5.9-jammy-slim
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/ubuntu/22.04/slim
 
-Tags: 5.9.2-focal-slim, 5.9-focal-slim, focal-slim
+Tags: 5.9.2-focal-slim, 5.9-focal-slim
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/ubuntu/20.04/slim
 
-Tags: 5.9.2-focal, 5.9-focal, focal
+Tags: 5.9.2-focal, 5.9-focal
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/ubuntu/20.04
 
-Tags: 5.9.2-amazonlinux2, 5.9-amazonlinux2, amazonlinux2
+Tags: 5.9.2-amazonlinux2, 5.9-amazonlinux2
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/amazonlinux/2
 
-Tags: 5.9.2-amazonlinux2-slim, 5.9-amazonlinux2-slim, amazonlinux2-slim
+Tags: 5.9.2-amazonlinux2-slim, 5.9-amazonlinux2-slim
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/amazonlinux/2/slim
 
-Tags: 5.9.2-centos7, 5.9-centos7, centos7
+Tags: 5.9.2-centos7, 5.9-centos7
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/centos/7
 
-Tags: 5.9.2-centos7-slim, 5.9-centos7-slim, centos7-slim
+Tags: 5.9.2-centos7-slim, 5.9-centos7-slim
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/centos/7/slim
 
-Tags: 5.9.2-rhel-ubi9, 5.9-rhel-ubi9, rhel-ubi9
+Tags: 5.9.2-rhel-ubi9, 5.9-rhel-ubi9
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/rhel-ubi/9
 
-Tags: 5.9.2-rhel-ubi9-slim, 5.9-rhel-ubi9-slim, rhel-ubi9-slim
+Tags: 5.9.2-rhel-ubi9-slim, 5.9-rhel-ubi9-slim
 Architectures: amd64, arm64v8
 GitCommit: 51f0941d5f6d92af9df8aa89fa75841f5036c127
 Directory: 5.9/rhel-ubi/9/slim
 
-Tags: 5.9.2-windowsservercore-ltsc2022, 5.9-windowsservercore-ltsc2022, windowsservercore-ltsc2022
+Tags: 5.9.2-windowsservercore-ltsc2022, 5.9-windowsservercore-ltsc2022
```

Comment:
Similar to https://github.com/docker-library/official-images/pull/3485#pullrequestreview-66902113, any particular reason to keep all the older `5.x` versions listed here (`5.2.x`-`5.8.x`)? Are they still supported? Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.

Reply:
Let me get back to you on this after we have a discussion with the Swift community, specifically the Swift Core team.